rules/windows/execution_windows_cmd_shell_susp_args.toml

```diff
@@ -65,76 +66,83 @@ tags = [
+        "*=wscri& set *", "*http*!COmpUternaME!*", "*start *.pdf * start /min cmd.exe /c *\\\\*", "*pip install*System.Net.WebClient*",
+        "*Invoke-WebReques*Start-Process*", "*-command (Invoke-webrequest*", "*copy /b *\\\\* ping *-n*", "*echo*.ToCharArray*"
+    ) or
+
+    (process.args : "echo" and process.parent.name : ("wscript.exe", "mshta.exe")) or
+    
+    process.args : ("1>?:\\*.vbs", "1>?:\\*.js") or
+
+    (process.args : "explorer.exe" and process.args : "type" and process.args : ">" and process.args : "start") or
+
+    (
+      process.parent.name : "explorer.exe" and
+      process.command_line : (
+        "*&&S^eT *",
+        "*&& set *&& set *&& set *&& set *&& set *&& call*",
+        "**\\u00??\\u00??\\u00??\\u00??\\u00??\\u00??\\u00??\\u00??*"
+      )
+    ) or
+
+    (process.parent.name : "explorer.exe" and process.args : "copy" and process.args : "&&" and process.args : "\\\\*@*\\*")
   ) and
 
   /* false positives */
   not (process.args : "%TEMP%\\Spiceworks\\*" and process.parent.name : "wmiprvse.exe") and
-  not process.parent.executable :
-                ("?:\\Perl64\\bin\\perl.exe",
-                 "?:\\Program Files\\nodejs\\node.exe",
-                 "?:\\Program Files\\HP\\RS\\pgsql\\bin\\pg_dumpall.exe",
-                 "?:\\Program Files (x86)\\PRTG Network Monitor\\64 bit\\PRTG Server.exe",
-                 "?:\\Program Files (x86)\\Spiceworks\\bin\\spiceworks-finder.exe",
-                 "?:\\Program Files (x86)\\Zuercher Suite\\production\\leds\\leds.exe",
-                 "?:\\Program Files\\Tripwire\\Agent\\Plugins\\twexec\\twexec.exe",
-                 "D:\\Agents\\?\\_work\\_tasks\\*\\SonarScanner.MSBuild.exe",
-                 "?:\\Program Files\\Microsoft VS Code\\Code.exe",
-                 "?:\\programmiweb\\NetBeans-*\\netbeans\\bin\\netbeans64.exe",
-                 "?:\\Program Files (x86)\\Public Safety Suite Professional\\production\\leds\\leds.exe",
-                 "?:\\Program Files (x86)\\Tier2Tickets\\button_gui.exe",
-                 "?:\\Program Files\\NetBeans-*\\netbeans\\bin\\netbeans*.exe",
-                 "?:\\Program Files (x86)\\Public Safety Suite Professional\\production\\leds\\leds.exe",
-                 "?:\\Program Files (x86)\\Tier2Tickets\\button_gui.exe",
-                 "?:\\Program Files (x86)\\Helpdesk Button\\button_gui.exe",
-                 "?:\\VTSPortable\\VTS\\jre\\bin\\javaw.exe",
-                 "?:\\Program Files\\Bot Framework Composer\\Bot Framework Composer.exe",
-                 "?:\\Program Files\\KMSYS Worldwide\\eQuate\\*\\SessionMgr.exe",
-                 "?:\\Program Files (x86)\\Craneware\\Pricing Analyzer\\Craneware.Pricing.Shell.exe",
-                 "?:\\Program Files (x86)\\jumpcloud-agent-app\\jumpcloud-agent-app.exe",
-                 "?:\\Program Files\\PostgreSQL\\*\\bin\\pg_dumpall.exe",
-                 "?:\\Program Files (x86)\\Vim\\vim*\\vimrun.exe") and
+  not process.parent.executable : (
```

Comment:
@w0rk3r this rule is important for threats-viz, maybe we can set `?process.parent.executable` as optional here and add CS too ?

Reply:
It would be very noise then, no?

Reply:
We could do a specific condition excluding by process.parent.name, would more FN prone but I think it is a good balance, wdyt?